locale/en_US/locale.po

```diff
@@ -1240,6 +1246,9 @@ msgstr "Identify reviewing interests (substantive areas and research methods):"
 msgid "user.register.form.emailConsentAllContexts"
 msgstr "Yes, I would like to be notified of new publications and announcements for all journals that I registered with."
 
+msgid "user.register.form.privacyConsentLabel"
+msgstr "Option to consent"
+
```

Comment:
All of the new locale keys in this PR are available in pkp-lib so they can be removed here.